docs/UsersGuide/source/index.rst

```diff
@@ -22,6 +22,7 @@ UFS Short-Range Weather App Users Guide
    InputOutputFiles
    LAMGrids
    ConfigWorkflow
+   DefineWorkflow
```

Comment:
@christinaholtNOAA It might make sense to add DefineWorkflow _after_ the RocotoInfo section.